ngraph/core/src/op/util/broadcast_base.cpp

```diff
@@ -361,14 +361,15 @@ bool op::util::BroadcastBase::evaluate(const HostTensorPtr& arg0,
                                        const HostTensorPtr& out,
                                        const AxisSet& broadcast_axes) const
 {
-    OV_ITT_SCOPED_TASK(itt::domains::nGraphOp, "op::util::BroadcastBase::evaluate<ET>");
-    runtime::reference::broadcast(arg0->get_data_ptr<const char>(),
-                                  out->get_data_ptr<char>(),
-                                  arg0->get_shape(),
-                                  out->get_shape(),
-                                  broadcast_axes,
-                                  arg0->get_element_type().size());
-    return true;
+    NGRAPH_OP_SCOPE(op_util_BroadcastBase_evaluate,
```

Comment:
Maybe use `util_BroadcastBase_evaluate` name since we already have `op` in `NGRAPH_OP_SCOPE`

Reply:
Fixed